src/components/UserProfile/BlueSquareLayout.jsx

```diff
@@ -42,7 +51,11 @@ const BlueSquareLayout = props => {
       ) : (
         <div>
           <p>BLUE SQUARES</p>
-          <BlueSquare blueSquares={infringements} handleBlueSquare={handleBlueSquare} role={role} />
+          <BlueSquare
+            blueSquares={userProfile?.infringments || userProfile?.infringments}
```

Comment:
Same as above. We just need to keep in mind that we will need change the spelling at some point.